lib/index.js

```diff
@@ -5,8 +5,8 @@
 'use strict';
 
 function html5_embed_renderer(tokens, idx, options, env, self, defaultRender) {
-  var audioRE = /^.*\.(ogg|mp3)$/gi;
-  var videoRE = /^.*\.(mp4|webm)$/gi;
+  var audioRE = /^.*\.(ogg|mp3|m4a|wav|ogg|oga|spx|flac|amr|3ga)$/gi;
+  var videoRE = /^.*\.(mp4|m4v|webm|3gp|3g2|ogv|ogm|avi)$/gi;
```

Comment:
Frankly, I don't know if we really need spx, amr, 3ga, 3gp, 3g2? Are they happen to be embedded anywhere?